test/Microsoft.Extensions.Configuration.Binder.Test/project.json

```diff
@@ -26,12 +26,11 @@
     },
     "net451": {
       "frameworkAssemblies": {
-        "System.Runtime": {
-          "type": "build"
-        },
-        "System.Threading.Tasks": {
-          "type": "build"
-        }
+        "System.Runtime": "",
```

Comment:
Not sure I just made this look like the other test projects via copy paste. This one was different than the rest.